src/icons/css/objects/_honeycomb.icons.objects.product-icons.scss

```diff
@@ -53,6 +53,7 @@
     }
 }
 
+.icon--schema-compare-mysql,
```

Comment:
Where's this alias being used please @philscott-rg ? Should we try and update it there, rather than add an alias for it here?

Reply:
The reason for adding an alias is that `schema-compare-mysql` is already set as the product slug in Wordpress, and I wanted to avoid what we ran into yesterday when I was fixing the same issue with pgnow: 
<img width="953" height="714" alt="image" src="https://github.com/user-attachments/assets/11e34261-ef64-4c68-a8a9-ec3f26edd256" />
If we update the slug in Wordpress then we'd need to update the website code to reflect that, so I think this works out simpler.